doc/signal_path.rst

```diff
@@ -96,29 +96,38 @@ phasors are done in single precision.
 Polyphase filter bank (PFB)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 A finite impulse response (FIR) filter is applied to the signal to condition
-the frequency-domain response. The filter is the product of a Hann window (to
-reduce spectral leakage) and a sinc (to broaden the peak to cover the
-frequency bin). Specifically, if there are :math:`n` output channels and
-:math:`t` taps in the polyphase filter bank, then the filter has length
+the frequency-domain response. The filter is the product of a window function
+Hann window (to reduce spectral leakage) and a sinc (to broaden the peak to
+cover the frequency bin). Specifically, if there are :math:`n` output channels
```

Comment:
"The filter is the product of a **window function Hann window** (...) and a since (...)."
* The snippet in bold doesn't sound quite right to me
* I'm not sure what to suggest as an alternative. Perhaps separate the sentences?

e.g.
> The filter is the product of a window function and a sinc. The window function defaults to a Hann window to reduce spectral leakage. The sinc is chosen to broaden the peak covering the frequency bin.

Reply:
I meant to delete "Hann window". See if you're happy with what's written now.